lms/djangoapps/labster_course_license/utils.py

```diff
@@ -61,7 +61,7 @@ def get_simulation_id(uri):
     """
     Returns Simulation id extracted from the passed URI.
     """
-    return urlparse(uri).path.strip('/').split('/')[-1]
+    return urlparse(uri.strip()).path.strip('/').split('/')[-1]
```

Comment:
Maybe we could consider using:
```
return urlparse(uri).path.strip(' /').split('/')[-1]
```
Which aside of being more concise, will possibly also more robust; if in the future we will allow for query parameters in the URL.

Otherwise :+1: